test/quantization/test_quantize_script.py

```diff
@@ -2461,14 +2461,16 @@ def __init__(self):
             def forward(self, x):
                 x = x + 5
                 return self.fc1(x)
-
-        m = torch.jit.script(M())
-        qconfig_dict = {'' : default_dynamic_qconfig}
-        model = quantize_dynamic_script(m, qconfig_dict)
-        # add op is not dynamically quantized.
-        FileCheck().check("aten::add") \
-                   .check("quantized::linear_dynamic") \
-                   .run(model.graph)
+        eager_model = M().eval()
+        x = torch.randn(5, 5)
+        for tracing in [True, False]:
+            model = get_script_module(eager_model, tracing, x)
+            qconfig_dict = {'' : default_dynamic_qconfig}
+            model = quantize_dynamic_script(model, qconfig_dict)
+            # add op is not dynamically quantized.
+            FileCheck().check("aten::add") \
```

Comment:
can you use `_test_op_impl`? It is in TestQuantizeScriptPTSQOps, but you can move it to QuantizationTestCase

Reply:
Sure. That currently only tests `quantize_script` API. I'll make it generic.

Reply:
or maybe define a `_test_op_impl` locally for this test

Reply:
@jerryzh168 refactored it a bit to make it usable by both.